docs/docs/css/extra.css

```
@@ -1,3 +1,18 @@
 h1, h2, h3 {
     margin-top: 77px;
 }
+
+#termynal {
+    height: 80ex !important; /* 40 lines of 2ex */
+    min-height: 80ex !important;
+    max-height: 80ex !important;
+    overflow: scroll !important;
+}
+
+[data-ty] {
+    line-height: 2ex !important; 
+}
+
```

```suggestion

.newline {
    line-height: 0 !important;
}

```
another kludge to get newlines in the terminal